src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,94 @@
+
+    public static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
+    private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
+
+
+    /**
+     * Returns true if the user input line should be ignored.
+     * Input should be ignored if it is parsed as a comment, is only whitespace, or is empty.
+     *
+     * @param rawInputLine full raw user input line.
+     * @return true if the entire user input line should be ignored.
+     */
+    public static boolean shouldIgnore(String rawInputLine) {
+        return rawInputLine.trim().isEmpty() || isCommentLine(rawInputLine);
+    }
+
+    /**
+     * Returns true if the user input line is a comment line.
+     *
+     * @param rawInputLine full raw user input line.
+     * @return true if input line is a comment.
+     */
+    private static boolean isCommentLine(String rawInputLine) {
+        return rawInputLine.trim().matches(COMMENT_LINE_FORMAT_REGEX);
+    }
+
+    /** Formats a list of strings as a viewable indexed list. */
+    public static String getIndexedListForViewing(List<String> listItems) {
+        final StringBuilder formatted = new StringBuilder();
+        int displayIndex = 0 + DISPLAYED_INDEX_OFFSET;
+        for (String listItem : listItems) {
+            formatted.append(getIndexedListItem(displayIndex, listItem)).append("\n");
+            displayIndex++;
+        }
+        return formatted.toString();
+    }
+
+    public static String getEnterCommandMessage() {
+        return LINE_PREFIX + "Enter command: ";
+    }
+
+    public static String getCommandEnteredMessage(String fullInputLine) {
+        return "[Command entered:" + fullInputLine + "]";
+    }
+
+    public static String[] getWelcomeMessage(String version, String storageFileInfo) {
+        return new String[]{DIVIDER,
+                DIVIDER,
+                MESSAGE_WELCOME,
+                version,
+                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
+                storageFileInfo,
+                DIVIDER};
+    }
+
+    public static String[] getInitFailedMessage() {
+        return new String[]{MESSAGE_INIT_FAILED, DIVIDER, DIVIDER};
+    }
+
+    public static String[] getGoodbyeMessage() {
+        return new String[]{MESSAGE_GOODBYE, DIVIDER, DIVIDER};
+    }
+
+    /**
+     * Formats a string as a viewable indexed list item.
+     *
+     * @param visibleIndex visible index for this listing
+     */
+    public static String getIndexedListItem(int visibleIndex, String listItem) {
+        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
+    }
+}
```

Comment:
This change does not seem to be related to the enhancement proposed by this PR. 
Try to limit to one LO per PR.